src/javaRosa.js

```diff
@@ -1743,6 +1747,17 @@ define([
                                 }
                                 if (value) {
                                     item.set(value, form, lang);
+                                    if (mmForms.hasOwnProperty(form) &&
+                                            !objectMap.hasOwnProperty(value)) {
+                                        // TODO make itext widgets display messages
+                                        // and then mug.addMessage(name, ...
+                                        mug.addMessage(null, {
+                                            key: "missing-multimedia-warning",
+                                            level: mug.WARNING,
+                                            message: "MultiMedia was not copied; " +
```

Comment:
Translation? (may be excluded as there isn't a neat way to mark translations in vellum code?)